src/webchat-ui/components/history/ChatScroller.tsx

```diff
@@ -73,7 +91,11 @@ export class ChatScroller extends React.Component<InnerProps, IState> {
     }
 
     componentDidUpdate(prevProps: InnerProps, prevState: IState, wasScrolledToBottom: boolean) {
-        if (wasScrolledToBottom) {
+        const { setScrollToPosition, scrollToPosition } = this.props;
+        if(scrollToPosition && setScrollToPosition) {
+            this.scrollToPosition(scrollToPosition);
```

Comment:
naming is a bit weird here. maybe change the name for the function to make sure that these two are not the same (not even same type). Examples:  onScrollToPosition or handleScrollToPosition

> this.scrollToPosition(scrollToPosition);

Reply:
Yes I agree. I think I could merge the two methods "scrollToBottom" and "scrollToPosition" in only one called "handleScrollTo", accepting an optional number parameter. If it is defined we scroll to that value, otherwise we follow the default logic to scroll to the bottom.